README.rst

```diff
@@ -38,8 +35,9 @@ learning projects:
 How DVC works
 =============
 
-We encourage you to read our `Get Started <https://dvc.org/doc/get-started>`_ guide to better understand what DVC
-is and how it can fit your scenarios.
+    We encourage you to read our `Get Started
+    <https://dvc.org/doc/get-started>`_ guides to better understand what DVC
+    does and how it can fit your scenarios.
```

Comment:
Not directly related but we should probably move this to AFTER the **Quick start** sections?

Reply:
- [ ] BTW it's missing info about Experiment Management (only covers Data Mgmt and Pipelines rn)

Reply:
Yes on both counts! It's on my personal to-do list along with a host of other readme improvements.

Reply:
https://github.com/iterative/dvc/pull/7917